src/server/rest/v1/service/UtilsService.ts

```diff
@@ -1418,6 +1457,39 @@ export default class UtilsService {
     return properties;
   }
 
+  public static async isAuthorizedOnChargingStation(tenant: Tenant, chargingStation: ChargingStation,
+      tagID: string, action: ServerAction, authAction: Action): Promise<{user: User, tag?: Tag}> {
+    return UtilsService.isTagIDAuthorizedOnChargingStation(tenant, chargingStation, null, tagID, action, authAction);
+  }
+
+  public static async isAuthorizedToStartTransaction(tenant: Tenant, chargingStation: ChargingStation,
+      tagID: string, transaction: Transaction, action: ServerAction, authAction?: Action): Promise<{user: User, tag?: Tag}> {
+    return UtilsService.isTagIDAuthorizedOnChargingStation(tenant, chargingStation, transaction, tagID, action, authAction);
+  }
+
+  public static async isAuthorizedToStopTransaction(tenant: Tenant, chargingStation: ChargingStation,
```

Comment:
[Low] Several method used in REST and OCPP services (2 different layers)
To be moved in new class in parent 'server' directory when used by several services?